mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_private_cloud_test.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The copyright is not needed, as it will be added automatically by default.

Reply:
Done.